roles/confluent.common/tasks/debian.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Gather OS Facts
+  setup:
+    filter: ansible_distribution_release
+    gather_subset:
```

Comment:
Might be worth while to add a comment stating this ignores all facts, and only gathers what is specified in the filter.